lib/transcriptomics/opa_plugin/requirements.txt

```
@@ -0,0 +1,2 @@
+candigv2-authx@git+https://github.com/CanDIG/candigv2-authx.git@v2.6.2 # CANDIG authorization wrapper
```

Should probably use latest version
```suggestion
candigv2-authx@git+https://github.com/CanDIG/candigv2-authx.git@v2.7.0 # CANDIG authorization wrapper
```